src/components/common/AppMenu.vue

```diff
@@ -68,6 +68,20 @@
         <i class="material-icons">chevron_right</i>
       </router-link>
 
+      <router-link
+        v-if="session.experimentalMode"
+        class="app-menu-item hide-xs"
+        to="/network"
+        exact="exact"
+        title="Network"
+        @click.native="close"
+      >
+        <h2 class="app-menu-title">
+          Network
```

Comment:
i think this should all be plural

`networks`

Reply:
Done